public/mergify-configuration-openapi.json

```diff
@@ -537,6 +537,12 @@
         "$ref": "#/definitions/BranchName"
       }
     },
+    "ReportModeArray": {
+      "$id": "/definitions/ReportModeArray",
+      "description": "list of report modes",
+      "type": "array",
+      "enum": ["check", "comment"]
```

Comment:
an array cannot be an enum, only a string can be in this case.
So you need to put `items` here and items must be type string and enum with the list
```
"items": {
  "type": "string",
  "enum": ["check", "comment"]
}
```

Reply:
Indeed good catch, I didn't knew how to put it, now I know!